server/views/pages/confirmedOnReport.njk

```diff
@@ -63,11 +62,13 @@
           'data-qa': 'confirmed-on-report-finish'
         }
       }) }}
-      </div>
     </div>
     <div class="print-only" data-qa="notification-of-being-on-report-section">
       {{ noticeOfBeingPlacedOnReport(noticeOfBeingPlacedOnReportData) }}
     </div>
   </div>
+  <div class="print-only">
+    {{ noticeOfBeingPlacedOnReport(noticeOfBeingPutOnReportData) }}
```

Comment:
This needs to be noticeOfBeingPlacedOnReportData

Reply:
Sorry it must have been the merge conflict I resolved (but wrongly)

Reply:
<div class="print-only" data-qa="notification-of-being-on-report-section">
      {{ noticeOfBeingPlacedOnReport(noticeOfBeingPlacedOnReportData) }}
    </div>
    
    is what's on main